pages/wizard/product_selection.py

```diff
@@ -44,15 +49,41 @@ def click_openshift(self):
         return self.openshift_checkbox.click()
 
     def select_products(self, products):
-        if 'rhv' in products:
+        if 'rhv' in products and not self.rhv_checkbox.is_selected():
```

Comment:
Added this because when you used the 'select_products' function twice, it un-selected the products. I see this as an unwanted behavior of this function. We can make 'unselect_products' maybe?